### ISSUE TITLE
[Request] SNES9X Core

### ISSUE THREAD

**Issue by doratos** [ClusterM/retroarch-clover#78]
Requesting add mainline core Snes9x for add Msu1 function.

**Comment by Rahzadan** [comment]
+1 on this, but not for MSU1 (not enough space for this on SNES Mini) but because it's MUCH newer than Snes9x 2010 and has many newer fixes!

**Comment by ClusterM** [comment]
> http://buildbot.libretro.com/nightly/linux/armhf/latest/
Are you talking about "snes9x_libretro.so.zip" there? Is it better that Snes 9x 2010?

**Comment by Rahzadan** [comment]
@ClusterM Yes, Snes9x 2010 is based off of SNES9x 1.52 which is 6+ years old at this point. snes9x_libretro.so.zip is based off Snes9x 1.54.1 which is still actively developed, has an all-new sound core (may even fix sound stuttering in some games, who knows) and has higher accuracy.

Would love to see it instead of (or in addition to) snes9x 2010.

Would also be curious to see if bsnes_mercury runs, as this is byuu's cycle accurate SNES emulator, although I expect that one may have some trouble on the NES/SNES mini.

**Comment by ClusterM** [comment]
It's not working on NES Mini for some reason :( Seems like it's linked to some missed libraries.

**Comment by astromatik** [comment]
Hello everyone
holder of the super mini nintendo, I am with great attention your discussion about a possible portage of the msu1 on our mini.
I am novice in computer science but if I can help you in whatever it will be with pleasure. beta test
the msu1 games are quite heavy in bytes, Mario is a 305 mb in zip and zelda format 550 mb...
Retroarch and the only way to emulate.bsnes performance and snes9x. What a joy to hear these music with the mini.
